### DISCUSSION TITLE
Switch function

### DISCUSSION THREAD

**Discussion by odem881** [serkri/SmartEVSE-3#249] [Q&A]
Hi, i have switch connected to SmartEVSE and have it set to switch from Solar To Smart. I must be doing something wrong because when i click the switch i get numbers displayed on the screen like it's counting when it will be started. What's up with this?

I just want to switch from different modes and not to set starting time. I expect that the screen will stay as it is, meaning it will still display the house and the power like before i click the switch.

**Comment by dingo35** [comment]
It is a safety feature to prevent the EVSE able/disable charging too fast, the counter shows you how much time to go before the safety time is over. 

It might very well be in that in the latest version there are less safety times, of course we try to get rid of them as much as possible without endangering your EV or your EVSE.

So please upgrade to latest version, we are now at 3.5.0-RC1 but on the verge of bringing out 3.5.0-RC2....
